editor/src/clj/editor/defold_project.clj

```diff
@@ -1876,6 +1879,7 @@
     (render-progress! (swap! progress progress/advance 4 (localization/message "progress.syncing-resources")))
     (du/log-time "Initial resource sync"
       (workspace/resource-sync! workspace-id [] (progress/nest-render-progress render-progress! @progress)))
+    (workspace/update-dependencies-min-version-from-cache! workspace-id)
```

Comment:
Here, we set project dependencies on lines 1876 or 1877, depending on internet availability, and the values may still be incorrect, but we will know about it later, after the sync. Introducing this "eventual correctness" seems unnecessary, because we already read the zips in `fetch-and-validate-libraries` where we locate, but currently don't read the `game.project` entry (`library-base-path`).